st2common/st2common/models/db/trigger.py

```diff
@@ -46,6 +46,7 @@ class TriggerTypeDB(stormbase.StormBaseDB,
     RESOURCE_TYPE = ResourceType.TRIGGER_TYPE
     UID_FIELDS = ['pack', 'name']
 
+    ref = me.StringField(required=False)
```

Comment:
I made it required=False so people upgrading don't have issues. At some point, we should make it required.